demisto_sdk/commands/common/timers.py

```diff
@@ -81,33 +131,32 @@ def report_time_measurements(group_name='Common', time_measurements_dir='time_me
         # sort by the total time
         list.sort(method_states, key=lambda method_stat: float(method_stat[2]), reverse=True)
 
-        write_mesure_to_logger(csv_data=method_states)
-        write_measure_to_file(time_measurements_dir=time_measurements_dir, group_name=group_name, csv_data=method_states)
+        write_mesure_to_logger(group_name, csv_data=method_states)
+        write_measure_to_file(time_measurements_dir=time_measurements_dir, group_name=group_name,
+                              csv_data=method_states)
 
     else:
         logger.debug(f'There is no timers registered for the group {group_name}')
 
 
-def write_mesure_to_logger(csv_data):
-
-    sentence = 'Time measurements stat'
-    output_msg = f"\n{Colors.Fg.cyan}{'#' * len(sentence)}\n"\
-        f"{sentence}\n"\
-        f"{'#' * len(sentence)}\n{Colors.reset}"
+def write_mesure_to_logger(name, csv_data, is_packs=False):
+    sentence = f'Time measurements stat for {name}'
+    output_msg = f"\n{Colors.Fg.cyan}{'#' * len(sentence)}\n" \
+                 f"{sentence}\n" \
+                 f"{'#' * len(sentence)}\n{Colors.reset}"
 
-    stat_info_table = tabulate(csv_data, headers=CSV_HEADERS)
+    stat_info_table = tabulate(csv_data, headers=CSV_HEADERS if not is_packs else PACK_CSV_HEADERS)
```

Comment:
what is the different?

Reply:
Two type of tables: one per function, and another for pack.

Reply:
so add it as a commnet